lib/Mollie/API/Resource/Customers/Subscriptions.rb

```diff
@@ -13,7 +13,7 @@ def getResourceObject
 
           def getResourceName
             customer_id = URI::encode(@parent_id)
-            "customers/#{parent_id}/subscriptions"
+            "customers/#{@parent_id}/subscriptions"
```

Comment:
Thanks for pointing this out! I think we should keep it consistent with the customer payments and mandates, which use `#{customer_id}`, the variable define the line above.